app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -640,6 +688,33 @@ class HomeFragment : Fragment() {
         navigateToSearch()
     }
 
+    private fun handleToolbarPasteAndGo(url: String) {
```

Comment:
Could this `handle` method be in the controller? I think the metrics logics should be there